### PR TITLE
Fix Test Notifications Ok button exception

### PR DESCRIPTION
**Pull request by andromeda-224**
### Prerequisites

- [ ] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Exception fixed by returning set of non-null PluginParameters in CreateParameters().
Dark themed Show/Hide Details text fixed with stylesheet.

### Alternate Designs
None.

### Why Should This Be In Core?
To fix the unexpected exception on the Ok button for Test Notifications.

### Benefits
To fix the unexpected exception on the Ok button for Test Notifications.

### Possible Drawbacks
None.

### Verification Process

1. Create a new Graph.
2. Select menu item Experimental -> Developer -> Test Notifications. 
3. Click OK.

Notification dialogs pop up and no exception is displayed. All text in the dialog boxes are readable.

### Applicable Issues
#2139

### CODE REVIEW

**Review by antares1470 — COMMENTED** [review]
I've had a look at the issue, and I wonder there may be a better way to address the problem.

Adding a plugin parameters object does solve the issue as but upon closer inspection, I actually question why there is a need to display a PluginParameterSwingDialog at all given there are no parameters (generally, it wouldn't make any sense to display one unless there were parameters to tinker with before executing). It seems like its adding an unnecessary extra layer. 

Acknowledging this is just a developer testing action but I wonder whether it would be better to simply remove the need for the dialog and instead just execute the plugin from the get go? This would also remove the need to have an empty parameters object.

**Review by Delphinus8821 — APPROVED** [review]
Agree with @antares1470 about removing the empty parameter object, otherwise looking good

**Review by Quasar985 — APPROVED** [review]

**Review by Delphinus8821 — APPROVED** [review]

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/TestNotificationsAction.java [hunk 1]
```diff
@@ -64,14 +64,8 @@ public TestNotificationsAction(final GraphNode context) {
     public void actionPerformed(final ActionEvent e) {
         // plugin notifications
         final Plugin plugin = new TestNotificationsPlugin();
-        final PluginParameters pp = plugin.createParameters();
-
-        final PluginParametersSwingDialog dialog = new PluginParametersSwingDialog(Bundle.CTL_TestNotificationsAction(), pp);
-        dialog.showAndWait();
-        if (PluginParametersDialog.OK.equals(dialog.getResult())) {
-            PluginExecution.withPlugin(plugin).withParameters(pp).executeLater(context.getGraph());
-        }
```
Comment (OrionsGuardian) [inline CoreTesting/src/au/gov/asd/tac/constellation/testing/TestNotificationsAction.java:73]: With the removal of this code, there are now some unused imports that should also be removed.

**Review by OrionsGuardian — CHANGES_REQUESTED** [review]

File: CoreTesting/src/au/gov/asd/tac/constellation/testing/TestNotificationsAction.java (hunk 1 above)
Comment (andromeda-224) [reply CoreTesting/src/au/gov/asd/tac/constellation/testing/TestNotificationsAction.java:73]: Done.

**Review by antares1470 — APPROVED** [review]

**Pull request merged**